src/content/developers/tutorials/Using-UseDApp-with-unity/index.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Using UseDApp with Unity
+description: Learn how to use UseDApp to interact with Unity engine project 
+author: "lucasespinosa28"
+tags: ["usedapp", "unity","react","dapp"]
+skill: intermediate
+lang: en
+sidebar: true
+sourceUrl: https://github.com/lucasespinosa28/usedappwithUnity
+---
+## Introduction {#introduction}
+In this tutorial, you will learn how to use [UseDApp](https://usedapp.readthedocs.io/en/latest/) a React plugins for rapid Dapp development, 
+and how to interact with a Unity using react, you will get your account balance inside a Unity and send a transaction from unity.
+
+### You need have [Unity](https://unity.com/) installed 
+
+## Unity project {#unity-project}
+On the Unity will have a text to receive the account balance, and a button to send a transaction using UseDApp,
+to communicate from unity to [UseDApp](https://usedapp.readthedocs.io/en/latest/) we will need an extra code using ```jslib```.
+
+### 1\. Create a new unity project and configure to webgl {#new-unity-project}
+Create a new unity project, the name of the project and if the project is 2d or 3d makes no difference. 
```

Comment:
Grammar check here please... and also, given it's a tutorial, we may want to just guide the user into 2D instead of leaving it up to them just for simplicity.  We can then remove the "makes no difference"